File: FrontEnd/src/app/features/reservations/manifest/classes/view-models/list/manifest-passenger-vm.ts
import { ManifestNationalityVM } from './manifest-nationality-vm'
import { ManifestPortVM } from './manifest-port-vm'
import { SimpleEntity } from 'src/app/shared/classes/simple-entity'

export interface ManifestPassengerVM {

    id: number
    refNo: string
    lastname: string
    firstname: string
    birthdate: string
    phones: string
    remarks: string
    specialCare: string
    gender: SimpleEntity
    nationality: ManifestNationalityVM
    port: ManifestPortVM

}

File: API/Features/Reservations/Manifest/ViewModels/List/ManifestPassengerVM.cs
using API.Infrastructure.Classes;

namespace API.Features.Reservations.Manifest {

    public class ManifestPassengerVM {

        public int Id { get; set; }
        public string RefNo { get; set; }
        public string Lastname { get; set; }
        public string Firstname { get; set; }
        public string Birthdate { get; set; }
        public string Phones { get; set; }
        public string Remarks { get; set; }
        public string SpecialCare { get; set; }
        public SimpleEntity Gender { get; set; }
        public ManifestNationalityVM Nationality { get; set; }
        public ManifestPortVM Port { get; set; }

    }

}